modules/installation/pages/uyuni-proxy-registration.adoc

```diff
@@ -0,0 +1,67 @@
+[[proxy-register]]
+= Proxy Registration
+
+Proxy systems are registered as Salt clients using a bootstrap script.
+
+This procedure describes software channel setup and registering the installed Proxy with an activation key as a {productname} client.
+
+[IMPORTANT]
+.Downloading Channels
+====
+Before you can select the correct child channels while creating the activation key, ensure you have created an {productname} Proxy 4 channel with all needed repositories properly synchronized.
+====
+
+[[proxy-register-procedure]]
+.Procedure: Registering the Proxy
+
+. Create an activation key based on your {productname} Proxy 4 channel.
+For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[Creating Activation Key].
++
+
+// .Proxy Activation Key
+// image::proxy-key.png[]
++
+////
```

Comment:
Shouldn't we remove this is (and other comments) if this doc is only for Uyuni?

Reply:
in general, yes.  But here we must check, which channel arrangement is actually needed (or the best).